### PR TITLE
update content-atom-model to 2.4.17

### PR DESCRIPTION
**Pull request by akash1810**
adds support for privacyStatus field

### CODE REVIEW

File: app/data/JsonConversions.scala [hunk 1]
```diff
@@ -11,6 +11,10 @@ object JsonConversions {
 
   implicit val category = Writes[Category](category => JsString(category.name.toLowerCase))
 
+  implicit val privacyStatusWrites: Writes[PrivacyStatus] = Writes[PrivacyStatus](privacyStatus => JsString(privacyStatus.name.toLowerCase))
+
+  implicit val privacyStatusReads: Reads[PrivacyStatus] = __.read[PrivacyStatus]
```
Comment (akash1810) [inline app/data/JsonConversions.scala]: hoping there's a nicer way to do this...

Comment (currysoup) [inline app/data/JsonConversions.scala]: For thrift enums I've been creating scala enums like this: https://github.com/guardian/media-atom-maker/blob/master/app/model/Category.scala

Comment (akash1810) [inline app/data/JsonConversions.scala]: oops, yeah forgot you mentioned that before. My bad!

**Comment by clloyd** [comment]
Looks good apart from broken build

**Comment by akash1810** [comment]
@clloyd made some changes, mind taking another 👀 ?

**Pull request merged**